api/controllers/CampaignBotController.js

```diff
@@ -241,6 +264,20 @@ class CampaignBotController {
    * @return {object}
    */
   loadCurrentSignup(req, id) {
+    this.debug(req, `loadCurrentSignup:${id}`);
+
+    if (this.isCommandClearCache(req)) {
+      this.debug(req, `isCommandClearCache`);
+
+      return app.locals.db.signups
+        .findByIdAndRemove(id)
```

Comment:
Not needed for merge, but we'll eventually want support for clearing your cached User model too (refs #636) -- I'm thinking a better approach may be to remove all Signup documents cached in the User's `campaigns` property upon deleting the User document -- clearing out the entire User cache instead of just the User's cache for this Campaign. That way we won't run into any Mongo errors of trying to create Signup documents with duplicate Signup id's.